openslides/utils/auth.py

```diff
@@ -65,7 +65,8 @@ def has_perm(self, perm, obj=None):
         """
         Checks if the user has a specific permission.
         """
-        return (perm in self.get_all_permissions())
+        default_group = CollectionElement.from_values('users/group', 1)
+        return perm in default_group.get_full_data()['permissions']
 
     def has_module_perms(self, app_label):
```

Comment:
Done. Our tests work, but this could be a problem in the future. Please add this code again before you complain about this PR.